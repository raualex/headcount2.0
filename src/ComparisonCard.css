.comparison-container {
  display: flex;
  justify-content: center;
}