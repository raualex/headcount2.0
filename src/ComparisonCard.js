import React, { Component } from 'react';
import './ComparisonCard.css';
import Card from './Card.js';
import PropTypes from 'prop-types';

class ComparisonCard extends Component {
  constructor(props) {
    super(props);

    this.state = {
      card1: props.card1,
      card2: props.card2
    };
  }

  render() {
    return (
      <div>
        <p>BOOOYAH</p>
      </div>
    );
  }
}

ComparisonCard.propTypes = {
  districtComparison: PropTypes.func.isRequired,
  card1: PropTypes.object,
  card2: PropTypes.object
};

export default ComparisonCard;