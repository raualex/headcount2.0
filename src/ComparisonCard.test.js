import React from 'react';
import { shallow } from 'enzyme';
import ComparisonCard from './ComparisonCard';

it('matches the snapshot', () => {
  const wrapper = shallow(<ComparisonCard 
    comparisonSchools={['COLORADO', 'ACADEMY 20']}
  />);

  expect(wrapper).toMatchSnapshot();
});

// it('prints cards dependent on props array', () => {
//   const wrapper = shallow(<ComparisonCard 
//     comparisonSchools={['COLORADO', 'ACADEMY 20']}
//   />);

//   expect(wrapper.find('.comparison-container').children()).toEqual(2)
// });