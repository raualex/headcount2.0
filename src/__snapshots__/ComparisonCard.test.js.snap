// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ComparisonCard
    comparisonSchools={
        Array [
            "COLORADO",
            "ACADEMY 20",
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
